maintainers/maintainer-list.nix

```diff
@@ -2267,6 +2267,12 @@
     githubId = 265220;
     name = "David Leung";
   };
+  DianaOlympos = {
```

Comment:
@DianaOlympos, please confirm or deny this is acceptable.

Reply:
Confirm, all good for me

Reply:
thanks!